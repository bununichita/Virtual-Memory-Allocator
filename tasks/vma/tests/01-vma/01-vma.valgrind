==36830== Memcheck, a memory error detector
==36830== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==36830== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==36830== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==36830== Parent PID: 36829
==36830== 
==36830== 
==36830== HEAP SUMMARY:
==36830==     in use at exit: 0 bytes in 0 blocks
==36830==   total heap usage: 10 allocs, 10 frees, 8,377 bytes allocated
==36830== 
==36830== All heap blocks were freed -- no leaks are possible
==36830== 
==36830== For lists of detected and suppressed errors, rerun with: -s
==36830== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
