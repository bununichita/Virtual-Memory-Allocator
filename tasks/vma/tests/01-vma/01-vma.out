Total memory: A bytes
Free memory: 9 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 1 - 2
Miniblock 1:	1	-	2	| RW-
Block 1 end
Total memory: A bytes
Free memory: 10 bytes
Number of allocated blocks: 0
Number of allocated miniblocks: 0
